flow/display_list_utils.h

```diff
@@ -314,80 +343,233 @@ class DisplayListBoundsCalculator final
+  static constexpr int kIsStrokedGeometry = 0x02;
+
+  // A geometric operation that may be a stroke or fill operation
+  // depending on the current state of the paint Style attribute.
+  // This flag will automatically assume |kApplyMaskFilter|.
+  static constexpr int kIsDrawnGeometry = 0x04;
+
+  static constexpr int kIsAnyGeometryMask =  //
+      kIsFilledGeometry |                    //
+      kIsStrokedGeometry |                   //
+      kIsDrawnGeometry;
+
+  // A geometric operation which has a path that might have
+  // end caps that are not rectilinear which means that square
+  // end caps might project further than half the stroke width
+  // from the geometry bounds.
+  // A rectilinear path such as |drawRect| will not have
+  // diagonal end caps. |drawLine| might have diagonal end
+  // caps depending on the angle of the line, and more likely
+  // |drawPath| will often have such end caps.
+  static constexpr int kGeometryMayHaveDiagonalEndCaps = 0x08;
+
+  // A geometric operation which has joined vertices that are
+  // not guaranteed to be smooth (angles of incoming and outgoing)
+  // segments at some joins may not have the same angle) or
+  // rectilinear (squares have right angles at the corners, but
+  // those corners will never extend past the bounding box of
+  // the geometry pre-transform).
+  // |drawRect|, |drawOval| and |drawRRect| all have well
+  // behaved joins, but |drawPath| might have joins that cause
+  // mitered extensions outside the pre-transformed bounding box.
+  static constexpr int kGeometryMayHaveProblematicJoins = 0x10;
+
+  // Some operations are inherently non-geometric and yet have the
+  // mask filter applied anyway.
+  // |drawImage| variants behave this way.
+  static constexpr int kApplyMaskFilter = 0x20;
+
+  // In very rare circumstances the ImageFilter is ignored.
+  // This is one of the few flags that turns off a step in
+  // estimating the bounds, rather than turning on any steps.
+  static constexpr int kIsUnfiltered = 0x40;
+
+  static constexpr SkScalar kMinStrokeWidth = 0.01;
+
+  skstd::optional<SkBlendMode> blend_mode_ = SkBlendMode::kSrcOver;
+  sk_sp<SkColorFilter> color_filter_;
 
-  void accumulateRect(const SkRect& rect, bool force_stroke = false);
+  SkScalar half_stroke_width_ = kMinStrokeWidth;
+  SkScalar miter_limit_ = 4.0;
+  int style_flag_ = kIsFilledGeometry;
+  bool join_is_miter_ = true;
+  bool cap_is_square_ = false;
+  sk_sp<SkImageFilter> image_filter_;
+  sk_sp<SkPathEffect> path_effect_;
+  sk_sp<SkMaskFilter> mask_filter_;
+  SkScalar mask_sigma_pad_ = 0.0;
+
+  bool paintNopsOnTransparenBlack();
+
+  static bool getFilteredBounds(SkRect& rect, SkImageFilter* filter);
+  bool adjustBoundsForPaint(SkRect& bounds, int flags);
+
+  void accumulateFlood();
+  void accumulateRect(const SkRect& rect, int flags) {
+    SkRect bounds = rect;
+    accumulateRect(bounds, flags);
+  }
+  void accumulateRect(SkRect& rect, int flags);
```

Comment:
Ditto on PascalCasing methods.

Reply:
I went back and forth on this in the original DL PR. I can revisit this, but I'm not going to do it in this PR.

Reply:
Many of these got either PascalCased or accessor_formatted in https://github.com/flutter/engine/pull/28753 per the style guide.